libbeat/docs/shared-feature-flags.asciidoc

```diff
@@ -34,6 +34,10 @@ Contains configuration for the FQDN reporting feature. When this feature is
 enabled, the fully-qualified domain name for the host is reported in the
 `host.name` field in events produced by {beatname_uc}.
 
+preview::[]
+
+NOTE: FQDN reporting is not currently supported in {endpoint-sec} or APM.
+
```

Comment:
Just to double check - and probably a silly question on my part - the reason we're including this note in the _Beats_ documentation is because APM server documentation is generated from these files as well?  In other words, this note would show up in APM server docs as well?

If so, I'm +1 to saying "APM Server" here but not sure if it makes sense to include Endpoint in here.

Reply:
Good point @ycombinator! The note about Endpoint Security and APM would appear only in the Beats docs, and it does seem out of context here, so I've removed it.